internal/helm/helm.go

```diff
@@ -75,6 +85,59 @@ func Remove(cluster *kubernetes.Cluster, logger logr.Logger, app models.AppRef)
 	return client.UninstallReleaseByName(names.ReleaseName(app.Name))
 }
 
+func RemoveService(cluster *kubernetes.Cluster, logger logr.Logger, app models.AppRef) error {
+	client, err := GetHelmClient(cluster.RestConfig, logger, app.Namespace)
+	if err != nil {
+		return err
+	}
+
+	return client.UninstallReleaseByName(names.ServiceReleaseName(app.Name))
+}
+
+func DeployService(logger logr.Logger, parameters ServiceParameters) error {
+	logger.Info("service helm setup", "parameters", parameters)
+
+	client, err := GetHelmClient(parameters.Cluster.RestConfig, logger, parameters.Namespace)
+	if err != nil {
+		return errors.Wrap(err, "create a helm client")
+	}
```

Comment:
Just for consistency I would keep the `logger` as the first parameter in the`RemoveService`, and returning a wrapped error, like in the `DeployService`: `errors.Wrap(err, "creating a helm client")`

Reply:
Will do.

Reply:
Done. Commit ef7781667a767d73ec06f9cfd85e9393830b8e54